save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/CreateOrganizationView.kt

```diff
@@ -165,14 +129,22 @@ class CreateOrganizationView : AbstractView<Props, OrganizationSaveViewState>(tr
                                 form {
                                     className = ClassName("needs-validation")
                                     div {
-                                        inputTextFormRequired(InputTypes.ORGANIZATION_NAME, state.isValidOrganizationName!!, "", "Organization name", true) {
-                                            changeFields(InputTypes.ORGANIZATION_NAME, it as Event)
+                                        inputTextFormRequired(
+                                            InputTypes.ORGANIZATION_NAME,
+                                            state.organizationDto.name.isEmpty() || state.organizationDto.validateName(),
+                                            "", "Organization name",
+                                            true,
+                                        ) {
+                                            setState {
+                                                organizationDto = organizationDto.copy(name = it.target.value)
+                                            }
                                         }
                                     }
                                     button {
                                         type = ButtonType.button
                                         className = ClassName("btn btn-info mt-4 mr-3")
                                         +"Create organization"
+                                        disabled = !state.organizationDto.validate()
```

Comment:
Any hints for a user that input data is invalid?

Reply:
My idea is to show errors on certain fields, not on the button. Guess for button it is ok

Reply:
For now those error messages are not that good, but later it will be fixed

Reply:
So every time you change a field of the object, two validations are calculated - field validation and the whole object validation, right? Maybe, we can sort of `isFormValid` state that would be updated from inside `inputTextFormRequired` if any of fields is invalid? And then this value can be used to disable the button.

Reply:
Decided to come back to this idea in phase 2 of this problem